### PR TITLE
Feature/tsd 2784

### PR DESCRIPTION
**Pull request by DanMad**
BIG change...

### CODE REVIEW

File: lib/accordion/accordion.test.tsx [hunk 1]
```diff
@@ -2,11 +2,13 @@ import React from 'react';
...
 import Accordion from './accordion';
 
+import { NAMESPACE } from '../utilities/ts/constants';
+
 import { mount } from 'enzyme';
 
 let wrapper;
 
-describe('<Acordion />', () => {
+describe(`<Acordion />`, () => {
```
Comment (andreamalvetta) [inline lib/accordion/accordion.test.tsx]: Typo <Acordion/> instead of <Accordion />

Comment (DanMad) [reply lib/accordion/accordion.test.tsx]: Hey bud, this one isn't right... It is 'accordion'.

Comment (DanMad) [reply lib/accordion/accordion.test.tsx]: OH! Now I understand...

Comment (andreamalvetta) [reply lib/accordion/accordion.test.tsx]: It should be more descriptive. ahha

**Pull request merged**